src/pages/CustomAlbum.jsx

```diff
@@ -0,0 +1,178 @@
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router";
+import styled from "styled-components";
+
+import ContentForm from "../components/ContentForm";
+import ContentSpectator from "../components/ContentSpectator";
+import { getAlbums, postAlbum } from "../api/customAlbum";
+
+const LENGTH = 7;
+
+const Container = styled.div`
+  display: flex;
+  flex-flow: row-reverse wrap;
+  justify-content: center;
+
+  .list {
+    flex-grow: 1;
+    justify-items: center;
+    max-width: 680px;
+  }
+`;
+
+function CustomAlbum() {
+  const { category } = useParams();
+  const [albums, setAlbums] = useState({
+    prevBuffer: [],
+    current: [],
+    nextBuffer: [],
+    prevPage: null,
+    nextPage: 1,
+  });
+
+  const handleSubmitForm = async function ({
+    date, heartCount, url, text,
+  }) {
+    await postAlbum(category, {
+      date, url, heartCount, text,
+    });
+  };
+
+  useEffect(async () => {
```

Comment:
useEffect에 들어가는 콜백함수는 반환값으로 뒷정리함수를 가져야하는데, 
async함수는 프로미스를 반환하기 때문에 콜백으로 바로 async함수를 넣으면 에러가 납니다! 
https://stackoverflow.com/questions/53332321/react-hook-warnings-for-async-function-in-useeffect-useeffect-function-must-ret
```suggestion
  useEffect(() => {
    async loadAlbums () {
```

Reply:
완전히 잊고 있었네요. 짚어주셔서 정말 감사합니다. 공유해주신 자료 확인후 다른 부분의 오류도 일괄 수정하였습니다.